networks/rpc/server.go

```diff
@@ -331,6 +337,7 @@ func (s *Server) handle(ctx context.Context, codec ServerCodec, req *serverReque
 				return codec.CreateErrorResponse(&req.id, &callbackError{err.Error()}), nil
 			}
 
+			*subCnt--
```

Comment:
Shouldn't this be ++?
```suggestion
			*subCnt++
```

Reply:
@winnie-byun Please check line 326, it is unsubscribe branch.